src/URLShortener.Services/Implementations/LinkService.cs

```diff
@@ -71,7 +71,6 @@ public async Task<Link> GetByIdAsync(GetByIdModel model, bool includeVisits = fa
     public async Task<Link> GetByShortAddressAsync(GetByShortAddressModel model)
     {
         var link = await _context.Links
-            .Where(l => l.UserId == model.UserId)
```

Comment:
We should consider adding AsNoTracking() here to speed up query, also avoiding unnecessary change tracking in entity framework.